### PR TITLE
PMM-8770 New MongoDB params

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

File: commands/inventory/add_agent_mongodb_exporter.go [hunk 1]
```diff
@@ -133,6 +138,10 @@ func init() {
 		StringVar(&AddAgentMongodbExporter.AuthenticationMechanism)
 	AddAgentMongodbExporterC.Flag("push-metrics", "Enables push metrics model flow,"+
 		" it will be sent to the server by an agent").BoolVar(&AddAgentMongodbExporter.PushMetrics)
-	AddAgentMongodbExporterC.Flag("disable-collectors",
-		"Comma-separated list of collector names to exclude from exporter").StringVar(&AddAgentMongodbExporter.DisableCollectors)
+
+	AddAgentMongodbExporterC.Flag("disable-collectors", "Comma-separated list of collector names to exclude from exporter").EnumsVar(
+		&AddAgentMongodbExporter.DisableCollectors, "diagnosticdata", "replicasetstatus", "dbstats", "topmetrics")
```
Comment (BupycHuk) [inline commands/inventory/add_agent_mongodb_exporter.go]: should we really mention all possible options here?

**Review by JiriCtvrtka — APPROVED** [review]

File: commands/inventory/add_agent_mongodb_exporter.go (hunk 1 above)
Comment (BupycHuk) [reply commands/inventory/add_agent_mongodb_exporter.go]: and as far as I see in this ticket it works different way. https://github.com/alecthomas/kingpin/pull/265
so it's not comma-separated list of collector names, you should repeat flag

Comment (percona-csalguero) [reply commands/inventory/add_agent_mongodb_exporter.go]: Reverted.

File: commands/inventory/add_agent_mongodb_exporter.go [hunk 2]
```diff
@@ -16,10 +16,9 @@
 package inventory
 
 import (
+	"github.com/percona/pmm-admin/commands"
 	"github.com/percona/pmm/api/inventorypb/json/client"
 	"github.com/percona/pmm/api/inventorypb/json/client/agents"
-
-	"github.com/percona/pmm-admin/commands"
```
Comment (BupycHuk) [inline commands/inventory/add_agent_mongodb_exporter.go]: please revert this change

File: commands/inventory/add_agent_mongodb_exporter.go [hunk 3]
```diff
@@ -97,6 +99,8 @@ func (cmd *addAgentMongodbExporterCommand) Run() (commands.Result, error) {
 			AuthenticationMechanism:       cmd.AuthenticationMechanism,
 			PushMetrics:                   cmd.PushMetrics,
 			DisableCollectors:             commands.ParseDisableCollectors(cmd.DisableCollectors),
+			StatsCollections:              cmd.StatsCollections,
```
Comment (BupycHuk) [inline commands/inventory/add_agent_mongodb_exporter.go]: can we parse string into []string here?

File: commands/inventory/add_agent_mongodb_exporter.go (hunk 2 above)
Comment (percona-csalguero) [reply commands/inventory/add_agent_mongodb_exporter.go]: Done

**Review by BupycHuk — APPROVED** [review]
